.github/workflows/publish-bicep.yaml

```diff
@@ -26,47 +26,13 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     env:
-      GOVER: '^1.21'
+      GOVER: "^1.21"
       GOPROXY: https://proxy.golang.org
     steps:
-      - uses: actions/checkout@v3
-      - name: Set up Go ${{ env.GOVER }}
-        uses: actions/setup-go@v4
-        with:
-          go-version: ${{ env.GOVER }}
-      - name: Setup npm
-        uses: actions/setup-node@v3
+      - name: Checkout PR branch
+        uses: actions/checkout@v2
```

Comment:
the edge branch seems to have v3, I am not sure I understand changing to v2, was it needed?

Reply:
Let's use v3 as it addresses the node 12 deprecation